### PR TITLE
feat: weather, user onboarding, random Wallpaper and Quote

### PR DESCRIPTION
**Pull request by maitrakhatri**
# features added
- detect live location (latitude and longitude) of the user
- use that live location to fetch weather details (temp, weather icon) and city name
- auto-update the location, temperature and weather icon without manual refresh and on manual refresh
- ask name when using the app first time, then save in local storage
- change wallpaper and quote on refresh

# thing to take care before review
- before you guys suggest use `axios` and `async await` for API calls, I already tried but then my code goes into an infinite loop. I'll work on it and update it ASAP
- Please ignore the node modules part, from next time I'll commit them separately.
- The code is bit messy, once it is approved I'll abstract the functions

# demo

https://user-images.githubusercontent.com/54719132/159647927-fab2b8e4-14a6-401c-bc4a-1be979e7ed76.mp4

### CODE REVIEW

File: gozo-browser-extension/src/index.css [hunk 1]
```diff
@@ -11,7 +11,7 @@ html, body, .app {
 }
 
 .app {
-    background-image: url("https://picsum.photos/1920/1080");
+    /* background-image: url("https://picsum.photos/1920/1080"); */
     display: flex;
```
Comment (surbhi84) [inline gozo-browser-extension/src/index.css:14]: comment can be removed if you don't need it here.

**Review by surbhi84 — APPROVED** [review]
The code looks neat.
Good use of react components.
The UI looks great.

File: gozo-browser-extension/src/index.css (hunk 1 above)
Comment (maitrakhatri) [reply gozo-browser-extension/src/index.css:14]: done

**Review by anupkgurung — APPROVED** [review]
Good to see auto update feature hope to see it live.
UI Seems good.

**Pull request merged**